README.md

```diff
@@ -13,6 +13,12 @@ cd TavernOfCho
 make init-full
 ```
 
+If you want to know all the option available with make, just run :
+
+```bash
+make help
```

Comment:
here you could just run make and it will also display the standard help (equivalent of make help) but it could be interesting to mention it

Reply:
I agree with you, will be fix on the last commit